src/config.ts

```diff
@@ -37,6 +37,11 @@ export interface SwaggerConfig {
    */
   entryFile: string;
 
+  /**
+   * Controllers glob path, contains directories with wild card to file names.
+   */
+  controllerPathGlobs?: string[];
```

Comment:
This is the addition that I'm confused by @joni7777. It doesn't make sense to have to have the user specific the same property in both the swagger config and the route config. It would be more ideal if both tsoa CLI commands looked in the same place. That's actually what I feel like must be happening since I am using the feature you wrote and I'm getting my routes and my swagger documents generated properly.

Reply:
But the swagger spec is running on the entry file only which don't include the controllers and the modules
Or does it have the reference using the import to the routes file?

I think we should also implement a way to consume tsoa on the app.ts file without pre generating routes file, by returning the routes class which being able to be used with
`app.use(TsoaRoutes)`

Reply:
Do you have slack/messenger or some IM app?

Reply:
I don’t have slack or any IM app. If tsoa isn’t generating the swagger and the routes then please describe the reproduction steps in the issue. I’m able to produce both using the new glob config.

Reply:
I'm trying to understand why there isn't any issue
If the swagger spec gets only the app entry file which is not importing the controllers
How does the spec metadata generator find all the controllers and the models if the routes generator find them by glob

Reply:
Maybe this is why it’s still working? Although I must say that I wasn’t aware that there was a “cache” until this other issue was  made: https://github.com/lukeautry/tsoa/issues/417

Reply:
I wrote an explanation in the issue #420 on why its working